2216/2/bad_words_analyzer.rb

```diff
@@ -0,0 +1,23 @@
+class BadWordsAnalyzer
+  def initialize(battle, words)
+    @battle = battle
+    @words = words
+  end
+
+  def analyze_bad
+    bad_words_count = @words.select do |word|
+      word.include?('*') || word.include?('(.') ||
+        RussianObscenity.obscene?(word)
+    end .size
+    [@battle.size, bad_words_count, bad_words_count / @battle.size,
+     @words.size.div(make_round_num)]
+  end
+
+  private
+
+  def make_round_num
```

Comment:
Why should the code scanning round number live in `BadWordsAnalyzer`?
I'd expect it to see in the `Battle` class.